doc/can-route.md

```diff
@@ -49,7 +49,7 @@ without changing the page.
 
 This provides the basics needed to
 create history enabled single-page apps.  However,
-`route` addresses several other needs aswell, such as:
+`route` addresses several other needs as well, such as:
 
   - Pretty urls.
   - Keeping routes independent of application code.
```

Comment:
Setup and teardown

Reply:
This already mentions setup and teardown of Widgets - line 57. Should we mention something else?

For your convenience here's the full list:
```md
  - Pretty urls.
  - Keeping routes independent of application code.
  - Listening to specific parts of the history changing.
  - Setup / Teardown of widgets.
```